proposals/0286-raise-block-limits-to-100M.md

```diff
@@ -0,0 +1,89 @@
+---
+simd: '0286'
+title: Increase Block Limits to 100M CUs
+authors:
+  - Lucas Bruder (Jito Labs)
+category: Standard
+type: Core
+status: Review
+created: 2025-05-20
+feature: TBD
+development:
+  - Anza - TBD
+  - Firedancer - TBD
+  - Jito Labs - TBD
+---
+
+## Summary
+
+Increase the block limit to 100M CUs.
+
+## Motivation
+
+Current block limits are set to 50M CUs. SIMD-0250 will increase them to 60M.
+Block limits' primary purpose is to ensure the vast majority of network
+participants are able to keep up with the network, by restricting the amount of
+work a leader is allowed to pack into a block. However, current mainnet traffic
+is largely not constrained by large block execution times. This proposal aims a
+substantial increase in block limits to 60M CUs, in order to provide additional
+capacity to the network.
+
+## New Terminology
+
+No new terms, but the following definitions are given for clarity:
+
+- Max Block Units - The maximum number of CUs that can be included in a block.
+- Max Writable Account Units - The maximum number of CUs that can be used to
+  write any given account in a block.
+- Max Vote Units - The maximum number of CUs that can be used by simple votes
+  in a block.
+- Max Block Accounts Data Size Delta - The maximum size of the accounts data
+  size delta in a block.
+
+## Detailed Design
+
+The following table shows the current block limits and the proposed block
+limits:
+
+| Type | Current Block Limit | SIMD-0250 Limit | Proposed Block Limit |
+|------|-----|---------------|---------------------|
+| Max Block Units | 50M | 60M | 100M |
+| Max Writable Account Units | 12M | 12M  | 12M |
```

Comment:
- research how frequently this has been hit
- suggest new number

Reply:
yes, I plan on filing a CU to increase this significantly